mkdocs.yml

```diff
@@ -73,6 +73,8 @@ nav:
         # - Declined:
   - Blog:
     - blog/index.md
+  - Contributing:
+    - Developer Guide: contributing/devguide.md
```

Comment:
should we put this under Guides?
I see https://network-policy-api.sigs.k8s.io/getting-started/ 
maybe contribting-guide should go there?